src/agent/sqlite_manager/tests/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+find_package(GTest REQUIRED)
+
+add_executable(test_SQLiteManager sqlite_manager_test.cpp)
```

Comment:
We forgot to do `configure_target(test_SQLiteManager)` here.

Verify that you have `clang-tidy-18` so you can get the checks and warnings locally too.

Reply:
Added